maintainers.html

```diff
@@ -0,0 +1,70 @@
+---
+layout: simple-page
+title: RIOT Maintainers
+subtitle: RIOT is nurtured by a strong community of maintainers.
+---
+<main>
+  <!-- ======= Maintainers Section ======= -->
+  <section id="maintainers">
+    <div class="container">
+      <p>
+      This is the current list of maintainers within the RIOT community with there area of
```

Comment:
The RIOT community is supported by the following maintainers. If a maintainer has expertise in certain parts of the [RIOT codebase](https://github.com/RIOT-OS/RIOT/) and has declared this within [CODEOWNERS](https://github.com/RIOT-OS/RIOT/blob/master/CODEOWNERS), this is explicitly noted.